### PR TITLE
Bug fix upon goMahjong messaging #106

### PR DESCRIPTION
**Pull request by Miles-Mom**
This is a bad bug.

## Pull Request Details

Your text here

## Contributor Agreement
- [x] I hold the rights to these contributions (if multiple rightsholders, please elaborate)
- [x] I understand that ecc521/mahjong is not an open-source project, and that the source code is not available under any open source license. 
- [x] I grant ecc521 (Tucker Willenborg) a non-revokable transferrable perpetual worldwide license to sublicense, redistribute, modify, build upon, commercialize, reuse, and/or distribute these contributions. 
- [x] This license is granted without any requirements or encumberments, absent attribution. I understand that no additional rights are granted to derivative works as a result of the integration of these contributions (ecc521/Tucker Willenborg will remain the sole rightsholder). 
- [x] This license is granted with no requirements for attribution (optional for larger contributions - please elaborate if requesting attribution)

### CODE REVIEW

**Comment by Miles-Mom** [comment]
The actual error happens on line 61 in goMahjong:

	this.messageAll([clientId], "roomActionGameplayAlert", {format: "%s went mahjong!", args:client.getNickname()}, {clientId, speech: "Mahjong"})
	
	this.setAllInstructions([clientId], {format: ["%s went mahjong!", "\n", "Press End Game to return everybody to the room screen. "], args:client.getNickname()})

	this.setInstructions(clientId, {format: ["%s went mahjong!", "\n", "Press End Game to return everybody to the room screen. "], argsI18n:"You"})   >>>>>>>>>>>>>>> this is the seemingly simple call that was a fix previously, I missed the hardcoded "You" earlier.

**Comment by Miles-Mom** [comment]
The actual error happens on line 61 in goMahjong:
Last one is the bad line,  the seemingly simple call that was a fix itself, when I missed the hardcoded "You" earlier. 

	this.messageAll([clientId], "roomActionGameplayAlert", {format: "%s went mahjong!", args:client.getNickname()}, {clientId, speech: "Mahjong"})
	this.setAllInstructions([clientId], {format: ["%s went mahjong!", "\n", "Press End Game to return everybody to the room screen. "], args:client.getNickname()})
	this.setInstructions(clientId, {format: ["%s went mahjong!", "\n", "Press End Game to return everybody to the room screen. "], argsI18n:"You"})

**Comment by Miles-Mom** [comment]
Also, the PR agreement text was kind of weird. I half expected to just check off the boxes, not editing the html file as a text, or revise the verbiage myself. Not sure if this is what you intended. I recall you asked earlier, could not find where/when now. FYI.

**Comment by ecc521** [comment]
@Miles-Mom 
I've adjusted the PR template. It should be much easier to fill out now.

File: server/Client.js [hunk 1]
```diff
@@ -53,14 +53,25 @@ class Client {
...
+			// eg: message(.., {format:"test %(param1)s, %(param2)s", args:{param1:Xxx}, argsI18n:{param2:Yyy-toBeLocalized}})
 			// note that mixing positional and named placeholders is not supported in sprintf therefore us
 			if (typeof message.argsI18n === "object") {
 				let argsI18n = message.argsI18n
 				for (let key in argsI18n) {
 					// save the localized data to the new combined args
 					args[key] = i18n.__({phrase:argsI18n[key], locale: this.locale})
 				}
+			} 
+			else if (typeof message.argsI18n === "string" ) {
```
Comment (ecc521) [inline server/Client.js:65]: It should be possible to cut out quite a few lines by moving the string detection upwards and converting into an object?

Anyways, no bit deal.

**Review by ecc521 — APPROVED** [review]

File: server/Client.js (hunk 1 above)
Comment (Miles-Mom) [reply server/Client.js:65]: got it. after this release (i don't trust my anti-midas touch). I am not available for the next hour or so. Will be back here then. Thanks Tucker!!!

**Pull request merged**